### PR TITLE
Feature/card

### PR DESCRIPTION
**Pull request by HanqingZhou**
Can use Card (base version), CardFigure, CardNews and CardPeople

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
-
+const Card: React.FC<CardProps> = ({
+    link,
+    header,
+    description,
+    image,
+}): JSX.Element => (
+    <article className='c-card' itemScope itemType='http://schema.org/Article'>
+        <a href={link} itemProp='url'>
+            <figure itemScope itemType='http://schema.org/ImageObject'>
```
Comment (marceloglacial) [inline src/components/Card/Card.tsx:23]: Please create a CardImage component to be shared to the other components.

File: src/components/Card/Card.tsx [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
 
 export interface CardProps {
-    type: 'base' | 'news' | 'video' | 'figure'
+    link: string
+    header: string
+    description?: string
+    image: CardImageProps
 }
 
 export interface CardImageProps {
```
Comment (marceloglacial) [inline src/components/Card/Card.tsx:10]: Mode this to the CardImage component

File: src/components/Card/Card.tsx [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
+    description,
+    image,
+}): JSX.Element => (
+    <article className='c-card' itemScope itemType='http://schema.org/Article'>
+        <a href={link} itemProp='url'>
+            <figure itemScope itemType='http://schema.org/ImageObject'>
+                <img src={image.src} alt={image.alt} loading='lazy' />
+            </figure>
+            <header>
+                <h3
```
Comment (marceloglacial) [inline src/components/Card/Card.tsx:27]: Please Create a CardTitle component to be shared to the other components

File: src/components/Card/Card.tsx [hunk 4]
```diff
@@ -1,34 +1,41 @@
...
+                    itemProp='name'
+                    dangerouslySetInnerHTML={{ __html: header }}
+                />
+                {description && (
+                    <p
```
Comment (marceloglacial) [inline src/components/Card/Card.tsx:32]: Please Create a CardDescription component to be shared to the other components

File: src/components/Card/CardVideo.tsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { CardImageProps } from '../Card'
+import { CardImageProps } from './Card'
 
 export interface CardVideoProps {
```
Comment (marceloglacial) [inline src/components/Card/CardVideo.tsx:4]: Please use the Icon component for the video icon. If its not there, added to the icon library.

**Review by marceloglacial — CHANGES_REQUESTED** [review]
Extra comments:

CardFigure:
- Is the description mandatory?
- Please check the dark mode version

Add the new Cards to the src/index.ts file to be exported to the react component library

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (HanqingZhou) [reply src/components/Card/Card.tsx]: Fixed, added CardImage component

File: src/components/Card/Card.tsx (hunk 2 above)
Comment (HanqingZhou) [reply src/components/Card/Card.tsx]: Fixed, added CardImage component

File: src/components/Card/Card.tsx (hunk 4 above)
Comment (HanqingZhou) [reply src/components/Card/Card.tsx]: Fixed,  added CardDescription component

File: src/components/Card/CardVideo.tsx (hunk 5 above)
Comment (HanqingZhou) [reply src/components/Card/CardVideo.tsx:6]: Fixed, added Icon component

File: src/components/Card/components/CardDescription.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    description: string
+}
+
+const CardDescription: FC<CardDescriptionProps> = ({
+    description,
+}): JSX.Element => {
+    return (
+        <p
+            itemProp='description'
+            dangerouslySetInnerHTML={{ __html: description }}
```
Comment (marceloglacial) [inline src/components/Card/components/CardDescription.tsx:13]: Now this is a new component, we should change dangerouslySetInnerHTML to children.

E.G.: `<p itemProp='description'>{children}</p>`

File: src/components/Card/components/CardTitle.tsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react'
+
+export interface CardTitleProps {
+    header: string
+}
+
+const CardTitle: FC<CardTitleProps> = ({ header }): JSX.Element => {
+    return <h3 itemProp='name' dangerouslySetInnerHTML={{ __html: header }} />
```
Comment (marceloglacial) [inline src/components/Card/components/CardTitle.tsx:8]: Now this is a new component, we should change dangerouslySetInnerHTML to children.

Same of description comment

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: src/components/Card/components/CardDescription.tsx (hunk 6 above)
Comment (HanqingZhou) [reply src/components/Card/components/CardDescription.tsx]: Fixed

File: src/components/Card/components/CardTitle.tsx (hunk 7 above)
Comment (HanqingZhou) [reply src/components/Card/components/CardTitle.tsx]: Fixed

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**